blueprints/ember-bootstrap/index.js

```diff
@@ -34,7 +34,7 @@ module.exports = {
   },
 
   beforeInstall(option) {
-    let bootstrapVersion = option.bootstrapVersion;
+    let bootstrapVersion = option.bootstrapVersion = parseInt(option.bootstrapVersion);
```

Comment:
I'm surprised this is necessary unless it's defensive. The docs and the blueprint insert it as a number. We should add the `radix`, as well.